docs/docs/learn/shards/coding-with-shards.md

```diff
@@ -6,42 +6,47 @@ In this chapter, we will be learning how to code with Shards so that you can wri
 
 A shard can take in data, process it, and output the results.
 
+![A shard takes in an input and produces an output.](assets/what-is-a-shard.png)
+
 There are many different types of data in the Shards language, and each shard will have specific data types that it can work with. 
 
 For example, the [`Math.Add`](../../../reference/shards/Math/Add/) shard can only work with numeric data types, while the [`Log`](../../../reference/shards/General/Log/) shard that prints information to the console can work with `Any` data type.
 
 Here are some of the data types found in Shards:
 
-| Data Type   | Description                                            | Example         |
-| :---------- | :----------------------------------------------------- | :-------------- |
-| `Any`       | Any data type.                                         |                 |
-| `Bool`      | Evaluates to either `true` or `false`.                 | `true`, `false` |
-| `Float`     | A numerical value with a decimal point.                | `2.53`, `9.124` |
-| `Int`       | A numerical value with no decimals. Read as "integer". | `2`, `9`        |
-| `None`      | No data type.                                          |                 |
-| `Sequence`  | A collection of values.                                | `[3, 25, 9]`    |
-| `String`    | Characters enclosed by double quotes.                  | "A string!"     |
-| `Wire`      | A sequence of shards.                                  |                 |
+| Data Type   | Description                                            | Example           |
+| :---------- | :----------------------------------------------------- | :---------------- |
+| `Any`       | Any data type.                                         |                   |
+| `None`      | No data type.                                          |                   |
+| `Bool`      | Evaluates to either `true` or `false`.                 | `true`, `false`   |
+| `Float`     | A numerical value with a decimal point.                | `2.53`, `-9.124`  |
+| `Int`       | A numerical value with no decimals. Read as "integer". | `2`, `-9`         |
+| `Sequence`  | A collection of values.                                | `[-3.2, "abc", 9]`|
+| `String`    | Characters enclosed by double quotes.                  | "A string!"       |
+| `Wire`      | A sequence of shards.                                  |                   |
+
+!!! note
+    The input and output of a shard can be made up of multiple data types. This is done by encapsulating the different types within a sequence. For example `["a string", 1.04, true]` is a sequence made up of a string, a float, and a boolean value.
```

Comment:
while it is true, it is already covered by the `Sequence` case. What I meant in my previous comment was different (see my last reply).